spec/integration/docker_rspec_helper.rb

```diff
@@ -0,0 +1,61 @@
+class RspecDocker
+  def initialize
```

Comment:
A new line would benefit readability here.

On the other side, (disclaimer I have zero experience with docker, so please educate me if I ask something silly). I wonder if the task in this constructor can actually fail for some reason? if this an option? or will they always run smoothly.

I ask this because in my experience having constructors with behaviour tend to be error prone and a wrong design, it's a lot better to have this as another procedure in the class. Makes sense what I'm saying in this case? please educate me if not :-1: